apollo-sample/build.gradle

```diff
@@ -29,8 +29,9 @@ dependencies {
   compile dep.appcompat
   compile dep.rxjava2
   compile dep.rxandroid
-  compile dep.retrofitRx2Adapter
   compile dep.okhttpLoggingInterceptor
+  compile project(path: ':apollo-runtime')
```

Comment:
might be outside the scope of this PR but should we also add this dependency through the Plugin?

It seems that apollo-runtime would have to be included for all use-cases.